snapcraft.yaml

```diff
@@ -71,6 +71,7 @@ parts:
     prime:
       - lib/libraft*so*
       - lib/*/libuv.so*
+      - lib/*/liblz4.so*
```

Comment:
@simondeziel why is this different compared to https://github.com/canonical/lxd-pkg-snap/pull/301/files#diff-e68b9afd67bb77dec8eda77796d3e0b784c7011203114996383eed6533be0cefR981 ?

Reply:
Thanks for catching this, it should have been the same. It is now.